setup.py

```diff
@@ -26,7 +26,7 @@
 def build_envd_if_not_found():
     if not os.path.exists("bin/envd"):
         logging.info("envd not found. Build from scratch")
-        errno = subprocess.call(["make", "build"])
+        errno = subprocess.call(["make", "build-local"])
```

Comment:
Should we use local? local sets git tag with the latest tag on GitHub, and make build follows the latest logic.

Reply:
Fixed. PTAL